client/state/themes/utils.js

```diff
@@ -242,3 +242,19 @@ export function getThemeTaxonomySlugs( theme, taxonomy ) {
 	const items = get( theme, [ 'taxonomies', taxonomy ], [] );
 	return items.map( ( { slug } ) => slug );
 }
+
+/**
+ * Returns the theme name extract from meta.
+ *
+ * @param  {string}  meta The purchase meta.
+ * @returns {string} The theme name.
+ */
+export function getThemeNameFromMeta( meta ) {
+	const hasBillingPeriod =
+		meta.lastIndexOf( '-monthly' ) > -1 || meta.lastIndexOf( '-yearly' ) > -1;
```

Comment:
Minor, but you could use `String.prototype.includes()` or `String.prototype.endsWith()` here instead of having to use the `-1` comparison and it might be more clear.

```suggestion
		meta.endsWith( '-monthly' ) || meta.endsWith( '-yearly' );
```